examples/mobile/UIComponents/css/style.css

```diff
@@ -468,7 +468,7 @@
 }
 
 /**************************************
-            Slider
+			Slider
```

Comment:
Wouldn't it be good to use 2 tabs?

Reply:
It has been changed by VSC :(

Reply:
In this file are different indentation for comments. I think it doesn't matter.